src/Client.php

```diff
@@ -144,7 +144,11 @@ public function sendMessage(Message $message): void
             throw new \RuntimeException(\sprintf('JSON encoding error %s: %s', \json_last_error(), \json_last_error_msg()));
         }
 
-        $this->httpClient->post('', [], $encoded);
+        $response = $this->httpClient->post('', [], $encoded);
+        $statusCode = $response->getStatusCode();
+        if ($statusCode !== 200) {
```

Comment:
- Yoda condition required.
- Also, a blank line before the `if` would be better.